operations/deployment/terraform/main.tf

```diff
@@ -24,7 +24,10 @@ resource "aws_instance" "server" {
   iam_instance_profile        = aws_iam_instance_profile.ec2_profile.name
 
   tags = {
-   Name = "${var.aws_resource_identifier} - Instance"
+    merge(
+      Name = "${var.aws_resource_identifier} - Instance",
+      var.additional_tags
+  )
```

Comment:
I think this merge can be removed because the additional tags are set in the `default_tags` here:
https://github.com/bitovi/github-actions-deploy-docker-to-ec2/pull/17/files#diff-088d9a58b5e5c54a030a57c8526a5a9659ede6fb83fb4e6a2eb7ade6f38600b0R39